### DISCUSSION TITLE
Create PV manually

### DISCUSSION THREAD

**Discussion by franrebo84** [rook/rook#8861] [Q&A] [answered]
I want to create a PV manually on ceph and call it from k8s.
Why I want this? I need to be able to name the block images and I cant find any other way.

Does anybody try this?

**Comment by leseb** [comment]
@Yuggupta27 @Madhu-1 PTAL

**Comment by Madhu-1** [comment, marked as answer]
Please check https://github.com/ceph/ceph-csi/blob/devel/docs/static-pvc.md to create static PVC and PV

**Reply by franrebo84** [reply]
Thanks! It worked like a charm.
